k-distribution/tests/regression-new/map-symbolic-tests-haskell/lookup-22-spec.k.out

```diff
@@ -3,20 +3,7 @@
   #Equals
     x
   } )
-#And
-  #Not ( {
-    X
-  #Equals
-    y
-  } )
```

Comment:
I think that removing this predicate is unsound. If `{ X #Equals y }`, then the original configuration is `#Bottom`, and we should not have applied the lookup rule.